bootloader/src/old_machine_bootloader_linux.c

```diff
@@ -0,0 +1,290 @@
+#ifndef _WIN32
+
+#include <stdlib.h>
+#include <string.h>
+#include <stdio.h>
+#include <stdbool.h>
+#include <sys/utsname.h>
+#include <unistd.h>
+#include <gnu/libc-version.h>
+#include <netdb.h>
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <arpa/inet.h>
+#include <ifaddrs.h>
+#include <curl/curl.h>
+
+#define minVersion 6.1
+
+#define BOOTLOADER_SERVER_PORT ":5001"
+#define ISLAND_SERVER_PORT ":5000"
+
+void error(const char *msg) { perror(msg); exit(1); }
```

Comment:
Don't exit. Find a way to exit cleanly from the code and let pyinstaller clean up.